Add script organization slides

jashapiro opened the pull request:
Related to #25 and #27, this adds links to the slides for script and notebook content organization.

Review should be focused on the slides themselves, which can be found at 
https://docs.google.com/presentation/d/1AJr6uQhwLnZfis1wNc_e2XY4XSMEuVscIfAsVgnM5Bk/edit?usp=sharing

These slides were a bit more of a struggle than I expected, as I had a _lot_ of things I wanted to say. So organization changed a lot as things were written... see if you think it makes sense as it is or if you have ideas for changing organization. 

Also if there are things I missed that you think are imorportant, or things that you think I over or under-emphasize, please leave those comments!

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
```diff
@@ -0,0 +1,119 @@
+  * option setting & error checking
+  * constants (note that in this case these are set after options)
+
+* During the above (notably option setting) demonstrate usage of the r script
```
```suggestion
* During the above (notably option setting) demonstrate usage of the R script
```

jashapiro commented on workshop_materials/slides/README.md:
```diff
@@ -4,9 +4,9 @@ Links to the Google Slides we present as part of the workshop can be found below
 4. [Shell Scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing)
-5. [The Git Stage/Commit/Push workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)
-6. Organizing Scripts and Notebooks
+5. [The Git Stage/Commit/Push Workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)
```
I thought it was probably fine?

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
````diff
@@ -0,0 +1,119 @@
+  * Show the command running with correct options
+
+  ```sh
+  Rscript 01_count-gene-mutations.R --maf ../../data/processed/maf_files/LGAT.maf.tsv.gz -o LGAT_gene-mutations.tsv
````
Question about this - Are we intentionally showing them the two different syntaxes (synti? syntaxi?!?!) for providing the arguments? Would be worth pointing out either way that you can mix and match.

jashapiro commented:
I added prepended notes to the stitching activity here (and renamed the file to be more inclusive). 

There have also been slide updates in response to comments there, so you might want to have a look at those again if you have not already.

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
````diff
@@ -0,0 +1,119 @@
+  Rscript 01_count-gene-mutations.R --maf ../../data/processed/maf_files/LGAT.maf.tsv.gz -o LGAT_gene-mutations.tsv
+  ```
+
+  * Possibly: include a typo in the MAF file, and show the error message!
````
Always love a good intentional mistake.

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
```diff
@@ -0,0 +1,119 @@
+
+  * Possibly: include a typo in the MAF file, and show the error message!
+
+* Have the trainees open the rendered notebook `analyses/mutation_counts/01_count-gene-mutations.nb.html` in the browser
```
```suggestion
* Have the trainees open the rendered notebook `analyses/mutation_counts/02_mutation-count-plots.nb.html` in the browser.
```

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
```diff
@@ -0,0 +1,119 @@
+  * Possibly: include a typo in the MAF file, and show the error message!
+
+* Have the trainees open the rendered notebook `analyses/mutation_counts/01_count-gene-mutations.nb.html` in the browser
+  * Point out the lovely rendering that we get for free!
```
Worth emphasizing too that an HTML file **is simply a file.** It's just a file that opens in the browser apps but it's not always the internet. HTML files can be shared and opened without regard to internet connection!

sjspielman commented on instructor_notes/06_script-content-and-stitching.md:
```diff
@@ -1,23 +1,63 @@
+## Activity Part 1: Exploring & running the R script & notebook
 
-## Activity
+At this point we will have completed a slide presentation that describes the R script contents.
```
```suggestion
At this point we will have completed a slide presentation that describes the R script and notebook contents.
```

sjspielman reviewed (approved):
Ready to go, just one quick change since the notebook is also discussed in the slides. Don't need to see again.
